### ISSUE TITLE
Select by fill colour not working (Sketch 68.2)

### ISSUE THREAD

**Issue by JBUXD** [wonderbit/sketch-select-similar-layers#38]
When running plugin action to **select similar layers by fill colour** in group, artboard or page - only the original selected layer is selected ("1 item selected in [page]").

**Comment by wonderbit** [comment]
Strange… do you have a file you can share? A blank sketch file with just the layers that are causing problems is fine.

**Comment by JBUXD** [comment]
Thanks for responding @wonderbit 

I'm actually seeing the issue in a brand new Sketch file as well. Here is an example I just tested it in...

[Select Similar Layers Plugin Issue.zip](https://github.com/wonderbit/sketch-select-similar-layers/files/5296175/Select.Similar.Layers.Plugin.Issue.zip)

**Comment by wonderbit** [comment]
It works on my machine 🤔 Do you have the latest version of the plugin (1.6)?

**Comment by JBUXD** [comment]
Hmm strange... Yep. Using the latest plugin. 

Any other troubleshooting tips? 

Currently, reorganising design system assets and without this plugin... 🤯

**Comment by wonderbit** [comment]
I have no idea… maybe there's a conflict with another plugin? I'm puzzled.

**Comment by JBUXD** [comment]
Yeah, I thought that might be the case. Is there anyway to look for a conflict with logs any other Sketch data output like that? Or will I need to manually find the conflicting plugin?

**Comment by wonderbit** [comment]
Try disabling all the other plugins just to see if that's the reason why it isn't working. I can't think of a reason why it would conflict with other plugins but 🤷🏻‍♂️

**Comment by hiyaneko** [comment]
Actually I'm having the same issue and nothing is working :(

**Comment by hiyaneko** [comment]
Tried disabling all plugins but this one and still not working

**Comment by wonderbit** [comment]
Weird… Could you try installing the dev tools plugin (https://github.com/skpm/sketch-dev-tools <https://github.com/skpm/sketch-dev-tools>) and see if any errors show up in the console?

**Comment by hiyaneko** [comment]
![Screenshot 2020-11-02 at 14 58 48](https://user-images.githubusercontent.com/32107480/97876321-f82db680-1d1b-11eb-9e65-1a35f79abf03.png)
This is the only thing that comes up !

**Comment by moniquep** [comment]
I have the same issue. I've tried disabling all plugins and testing, but the confirmation message says "1 item selected in page". Happens to all color actions including font and border color.

**Comment by fengyiyi11** [comment]
Same issue
